pkg/services/oracle/network.go

```diff
@@ -86,10 +86,9 @@ func getDefaultClient(cfg config.OracleConfiguration) *http.Client {
 		DialContext: d.DialContext,
 	}
 	client.Timeout = cfg.RequestTimeout
+	// Override default CheckRedirect behaviour so that there's no restriction on the
+	// number of redirects (10 at max is allowed by default).
```

Comment:
Isn't it `AllowAutoRedirect = false` which means no redirections allowed?

Reply:
It is, but at the same time redirections are handled by the C# node code itself and this code doesn't limit the number of redirections. So technically they are allowed.

Reply:
Wait, what?! It can't be unlimited!

Reply:
Resolved, limited by 2 including.